libs/books/feature/src/lib/book-search/book-search.component.scss

```diff
@@ -68,5 +68,11 @@ form {
   justify-content: center;
   min-height: 50vh;
   font-size: 1.5rem;
-  color: $gray40;
+  color: $gray80;
+}
+
+button {
+  background: none;
+  border: none;
+  color: black;
```

Comment:
Instead of hardcoding color name here, use variables defined already in code just like line no 71